CafeApp/database/productsDB.tsx

```diff
@@ -0,0 +1,73 @@
+// This is our products database for now. Later we will replace this with database and redux state
```

Comment:
.tsx file extension should be used for files that returns jsx compoents. This file is purely for saving some object so could have .ts extension